src/js/base/DialogContent.tsx

```diff
@@ -22,7 +22,7 @@ export const DialogContent = styled(DialogPrimitive.Content)<DialogContentProps>
     border-radius: ${props => props.theme.borderRadius.md};
     box-shadow: ${props => props.theme.boxShadow.lg};
     position: fixed;
-    top: 40%;
+    top: 50%;
```

Comment:
Hmm this works, but the original looks better with some of the other dialogues. (create user for example) Take a look to see if you can find a way to have the position adjust with the size of the window. (vh + vw could be handy) Tricky thing here could be dealing with the fact that the base component doesn't know what height the final modal will be